### ISSUE TITLE
Centos 7 __glib c issue

### ISSUE THREAD

**Issue by jonmmease** [conda-forge/kaleido-core-feedstock#4]
Making an issue from https://github.com/conda-forge/kaleido-core-feedstock/pull/2#issuecomment-747751614

> @bandersen23: @jonmmease Does this prevent CentOS 7 installs or am I misunderstanding that? I was receiving an Unsatisfiable error for "__glibc >=2.17,<3.0.a0", when trying to install the Linux version. I'm trying to understand why that is +1 Thank you!

> @jonmmease: No, CentOS 7 should satisfy __glibc >=2.17 (https://centos.pkgs.org/7/centos-x86_64/glibc-2.17-317.el7.x86_64.rpm.html). Is this an x64 installation? FWIW, the ARM build isn't on conda-forge.

> @bandersen23: That was what I thought. I do have an x64 install. I'm suspicious that the feedstock replaced sysroot_linux-64 with __glibc in the index.json. If you download the linux64 tar from Anaconda and look at the index.json file packaged up with it, you'll see that it will cause conda to look for __glibc rather than sysroot linux-64. sysroot_linux-64 resolves in conda, but __glibc will not (because it should be in the sysroot package referenced earlier). Does that make sense to you? I'm at least thinking that is the problem.
>
> I hope you know that I don't think this PR is the root of the problem as it makes sense now that I've read it all, I just think something isn't resolving correctly. If this is the case, I'd assume there's a problem in how conda built the package, so it might be a simple fix to rebuild, but I'd defer to the conda-forge admin team if that's even possible +1

**Comment by jonmmease** [comment]
I haven't been able to reproduce this using a [CentOS 7 docker image](https://hub.docker.com/layers/centos/library/centos/7/images/sha256-43964203bf5d7fe38c6fca6166ac89e4c095e2b0c0a28f6c7c678a1348ddc7fa?context=explore).

From this image

```
$ docker run -it centos:7 bash 
$ yum install wget
$ wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
$ chmod +x ./Miniconda3-latest-Linux-x86_64.sh
$ ./Miniconda3-latest-Linux-x86_64.sh
$ source ~/.bashrc
$ conda install -c conda-forge kaleido-core
```
```
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /root/miniconda3

  added / updated specs:
    - kaleido-core


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    _libgcc_mutex-0.1          |      conda_forge           3 KB  conda-forge
    _openmp_mutex-4.5          |            1_gnu          22 KB  conda-forge
    ca-certificates-2020.12.5  |       ha878542_0         137 KB  conda-forge
    certifi-2020.12.5          |   py38h578d9bd_0         143 KB  conda-forge
    conda-4.9.2                |   py38h578d9bd_0         3.0 MB  conda-forge
    expat-2.2.9                |       he1b5a44_2         191 KB  conda-forge
    font-ttf-dejavu-sans-mono-2.37|       hab24e00_0         388 KB  conda-forge
    font-ttf-inconsolata-2.001 |       hab24e00_0          97 KB  conda-forge
    font-ttf-source-code-pro-2.030|       hab24e00_0         927 KB  conda-forge
    font-ttf-ubuntu-0.83       |       hab24e00_0         1.9 MB  conda-forge
    fontconfig-2.13.1          |    h736d332_1003         319 KB  conda-forge
    fonts-conda-forge-1        |                0           4 KB  conda-forge
    freetype-2.10.4            |       h7ca028e_0         912 KB  conda-forge
    icu-68.1                   |       h58526e2_0        13.0 MB  conda-forge
    kaleido-core-0.1.0         |       h3644ca4_0        56.3 MB  conda-forge
    libgcc-ng-9.3.0            |      h5dbcf3e_17         7.8 MB  conda-forge
    libgomp-9.3.0              |      h5dbcf3e_17         378 KB  conda-forge
    libiconv-1.16              |       h516909a_0         1.4 MB  conda-forge
    libpng-1.6.37              |       h21135ba_2         306 KB  conda-forge
    libstdcxx-ng-9.3.0         |      h2ae2ef3_17         4.0 MB  conda-forge
    libuuid-2.32.1             |    h7f98852_1000          28 KB  conda-forge
    libxml2-2.9.10             |       h72842e0_3         1.3 MB  conda-forge
    nspr-4.29                  |       he1b5a44_1         232 KB  conda-forge
    nss-3.60                   |       hb5efdd6_0         2.1 MB  conda-forge
    openssl-1.1.1i             |       h7f98852_0         2.1 MB  conda-forge
    python_abi-3.8             |           1_cp38           4 KB  conda-forge
    sqlite-3.34.0              |       h74cdb3f_0         1.4 MB  conda-forge
    ------------------------------------------------------------
                                           Total:        98.3 MB

The following NEW packages will be INSTALLED:

  _openmp_mutex      conda-forge/linux-64::_openmp_mutex-4.5-1_gnu
  expat              conda-forge/linux-64::expat-2.2.9-he1b5a44_2
  font-ttf-dejavu-s~ conda-forge/noarch::font-ttf-dejavu-sans-mono-2.37-hab24e00_0
  font-ttf-inconsol~ conda-forge/noarch::font-ttf-inconsolata-2.001-hab24e00_0
  font-ttf-source-c~ conda-forge/noarch::font-ttf-source-code-pro-2.030-hab24e00_0
  font-ttf-ubuntu    conda-forge/noarch::font-ttf-ubuntu-0.83-hab24e00_0
  fontconfig         conda-forge/linux-64::fontconfig-2.13.1-h736d332_1003
  fonts-conda-forge  conda-forge/noarch::fonts-conda-forge-1-0
  freetype           conda-forge/linux-64::freetype-2.10.4-h7ca028e_0
  icu                conda-forge/linux-64::icu-68.1-h58526e2_0
  kaleido-core       conda-forge/linux-64::kaleido-core-0.1.0-h3644ca4_0
  libgomp            conda-forge/linux-64::libgomp-9.3.0-h5dbcf3e_17
  libiconv           conda-forge/linux-64::libiconv-1.16-h516909a_0
  libpng             conda-forge/linux-64::libpng-1.6.37-h21135ba_2
  libuuid            conda-forge/linux-64::libuuid-2.32.1-h7f98852_1000
  libxml2            conda-forge/linux-64::libxml2-2.9.10-h72842e0_3
  nspr               conda-forge/linux-64::nspr-4.29-he1b5a44_1
  nss                conda-forge/linux-64::nss-3.60-hb5efdd6_0
  python_abi         conda-forge/linux-64::python_abi-3.8-1_cp38

The following packages will be UPDATED:

  ca-certificates    pkgs/main::ca-certificates-2020.10.14~ --> conda-forge::ca-certificates-2020.12.5-ha878542_0
  certifi            pkgs/main/noarch::certifi-2020.6.20-p~ --> conda-forge/linux-64::certifi-2020.12.5-py38h578d9bd_0
  libgcc-ng           pkgs/main::libgcc-ng-9.1.0-hdf63c60_0 --> conda-forge::libgcc-ng-9.3.0-h5dbcf3e_17
  libstdcxx-ng       pkgs/main::libstdcxx-ng-9.1.0-hdf63c6~ --> conda-forge::libstdcxx-ng-9.3.0-h2ae2ef3_17
  openssl              pkgs/main::openssl-1.1.1h-h7b6447c_0 --> conda-forge::openssl-1.1.1i-h7f98852_0
  sqlite                pkgs/main::sqlite-3.33.0-h62c20be_0 --> conda-forge::sqlite-3.34.0-h74cdb3f_0

The following packages will be SUPERSEDED by a higher-priority channel:

  _libgcc_mutex           pkgs/main::_libgcc_mutex-0.1-main --> conda-forge::_libgcc_mutex-0.1-conda_forge
  conda               pkgs/main::conda-4.9.2-py38h06a4308_0 --> conda-forge::conda-4.9.2-py38h578d9bd_0


Proceed ([y]/n)? 
```

@bandersen23. Could you include the version of conda that you're using?  I think it would be worth trying to update it.

**Comment by bandersen23** [comment]
@jonmmease I was able to successfully install this using the steps above in Docker. Updating `conda` should fix it, as I had Miniconda 4.7.10 installed. When I updated to Miniconda 4.8.2, everything worked as expected (as well as the latest version as well). Good to know!

**Miniconda 4.7.10 image (errors out)** 
```bash
$ docker run -it centos:7 bash 
$ yum install wget
$ wget https://repo.anaconda.com/miniconda/Miniconda3-4.7.10-Linux-x86_64.sh
$ chmod +x ./Miniconda3-4.7.10-Linux-x86_64.sh
$ ./Miniconda3-4.7.10-Linux-x86_64.sh
$ source ~/.bashrc
$ conda create -n kaleido
$ conda activate kaleido
$ conda install -c conda-forge kaleido-core
```

**Miniconda 4.8.2 image (successful)**
```bash
$ docker run -it centos:7 bash 
$ yum install wget
$ wget https://repo.anaconda.com/miniconda/Miniconda3-py37_4.8.2-Linux-x86_64.sh
$ chmod +x ./Miniconda3-py37_4.8.2-Linux-x86_64.sh
$ ./Miniconda3-py37_4.8.2-Linux-x86_64.sh
$ source ~/.bashrc
$ conda install -c conda-forge kaleido-core
```

Thanks for working with me on this. Have a happy holiday!

**Comment by jonmmease** [comment]
Great!  Thanks for letting us know it's working with an updated version of conda. Happy Holidays :slightly_smiling_face:
